ethdb/pebble/pebble.go

```diff
@@ -518,17 +520,17 @@ type batch struct {
 }
 
 // Put inserts the given value into the batch for later committing.
-func (b *batch) Put(key, value []byte) error {
-	b.b.Set(key, value, nil)
+func (b *batch) Put(key, value []byte) (err error) {
+	err = b.b.Set(key, value, nil)
 	b.size += len(key) + len(value)
-	return nil
+	return
```

Comment:
```suggestion
	return err
```
Yes I know it can be done like you did it, but we generally don't do it that way. 
It's ok to declare the outputs if you want to make it obvious what each return means, for example if the return is `bool, bool, bool`, it's nicer if it says `exists, foo, existedEarlier bool`. 

However, that doesn't apply here. So just do 
```
err := b.b....
b.size ...
return err
```

Reply:
Will change it.